src/index.js

```diff
@@ -58,10 +60,65 @@ function buildTaskLeftElement(task) {
   return taskLeftElement;
 }
 
-function buildTaskRightElement() {
+function buildTaskRightElement(task) {
   const taskRightElement = createDivWithClass('todo-list-task-right');
-  taskRightElement.innerHTML = '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>';
+  const editElement = createElementWithClass('i', 'fa fa-ellipsis-v');
+  editElement.setAttribute('id', `edit-task-${task.index}`);
+  editElement.setAttribute('aria-hidden', 'true');
+  editElement.addEventListener('click', (e) => {
+    const targetElement = e.target;
+    const { parentElement } = targetElement;
+    const taskElement = parentElement.parentElement;
+
+    const taskIndex = targetElement.id.substring('edit-task-'.length);
+    const deleteElement = document.getElementById(`delete-task-${taskIndex}`);
+
+    taskElement.classList.add('editable');
+    targetElement.classList.add('hidden');
+    deleteElement.classList.remove('hidden');
+
+    const tasks = loadTasks();
+    const editableTask = tasks.filter((task) => task.index == taskIndex )[0]; /* eslint-disable-line  */
+
+    const taskDescriptionInputElement = createElement('input');
+    taskDescriptionInputElement.setAttribute('type', 'text');
+    taskDescriptionInputElement.setAttribute('class', 'editable');
+    taskDescriptionInputElement.setAttribute('value', editableTask.description);
+    taskDescriptionInputElement.id = `task-description-${taskIndex}`;
+
+    const taskDescriptionElement = document.getElementById(`todo-list-task-description-${taskIndex}`);
+    const taskLeftElement = taskDescriptionElement.parentElement;
+
+    /* Remove description and add new input */
+    taskLeftElement.removeChild(taskDescriptionElement);
+    taskLeftElement.appendChild(taskDescriptionInputElement);
+  });
+
+  const deleteElement = createElementWithClass('i', 'fa fa-trash hidden');
+  deleteElement.setAttribute('id', `delete-task-${task.index}`);
+  deleteElement.setAttribute('aria-hidden', 'true');
+  deleteElement.addEventListener('click', (e) => {
+    const taskIndex = e.target.id.substring('delete-task-'.length);
+
+    let tasks = loadTasks();
+    tasks = tasks.filter((task) => task.index != taskIndex); /* eslint-disable-line eqeqeq */
```

Comment:
- [ ] Please avoid unnecessary linter disabling. Here, for example, you can use `!==` by converting the data type of the variables. Please read this article on how strict comparisons and variable types work, to figure out how to do this: https://learn.onemonth.com/difference-between-equal-signs-javascript/

Reply:
Done.